app/controllers/tours_controller.rb

```diff
@@ -14,5 +14,8 @@ def show
     @reviews = @tour.reviews.paginate(page: params[:page])
     @review = Review.new
     @comment = Comment.new
+    @reviews_likes = current_user.likes.where(likeable_type: "Review")
```

Comment:
phải thêm điều kiện where theo tour_id nữa chứ 
ko thì sẽ lấy hết toàn bộ like của db ra